sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/package.scala

```diff
@@ -100,6 +70,19 @@ package object util extends Logging {
     file
   }
 
+  private def writeToByteArrayOutputStream(inStream: InputStream): ByteArrayOutputStream =
+    Utils.tryWithResource(new ByteArrayOutputStream) { outStream =>
```

Comment:
This no longer closes the inStream - I think that's good to keep

Reply:
It is my bad, let me revert this

Reply:
[3307d28](https://github.com/apache/spark/pull/35622/commits/3307d283798010452f35e41adc1ce374e9508355) fix this